app.rb

```diff
@@ -0,0 +1,116 @@
+require_relative('./person')
+require_relative('./book')
+require_relative('./rental')
+require_relative('./student')
+require_relative('./teacher')
+require 'date'
+
+class Methods
+  def initialize
+    @rentals = []
+    @people = []
+    @books = []
+  end
+
+  def list_books
+    puts 'No books available' if @books.empty?
+    @books.each { |book| puts book }
+  end
+
+  def list_people
+    puts 'No people available' if @people.empty?
+    @people.each { |person| puts person }
+  end
```

Comment:
- [ ] <b>REQUIRED:</b> Your listing is still not working as expected. It is listing class objects, not data. Kindly fix it.

![scrnli_4_9_2022_2-53-57 AM](https://user-images.githubusercontent.com/45217364/162550309-df2fe9a3-b4b1-4e2f-9241-7708e4fba74e.png)

Reply:
Done!